Assets/MixedRealityToolkit.Tests/EditModeTests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -16,7 +17,7 @@ public class TestFixture_01_MixedRealityToolkitTests
         [Test]
         public void Test_01_InitializeMixedRealityToolkit()
         {
-            TestUtilities.CreateScenes();
+            TestUtilities.EditorCreateScenes();
```

Comment:
This looks unrelated to the original change (Interactable tests). Did you intend to make the change here?

Reply:
I wanted to differentiate clearly between play mode setup and TestUtilities.CreateScenes, which is strictly for editor tests.